content/docs/forms.md

```diff
@@ -23,13 +23,13 @@ HTML form elements work a little bit differently from other DOM elements in Reac
 
 This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
 
-## Controlled Components {#controlled-components}
+## Kontrollü Componentler {#controlled-components}
 
-In HTML, form elements such as `<input>`, `<textarea>`, and `<select>` typically maintain their own state and update it based on user input. In React, mutable state is typically kept in the state property of components, and only updated with [`setState()`](/docs/react-component.html#setstate).
+HTML’de, `<input>`, `<textarea>` ve `<select>` gibi form elemanları genellikle kendi state’ini korur ve kullanıcı girdisine dayalı olarak güncelleşir. React’te ise state’ler genellikle componentlerin this.state özelliğinde saklanır ve yalnızca  [`setState()`](/docs/react-component.html#setstate). ile güncellenir.
 
-We can combine the two by making the React state be the "single source of truth". Then the React component that renders a form also controls what happens in that form on subsequent user input. An input form element whose value is controlled by React in this way is called a "controlled component".
+React state’te tek kaynak olarak ikisini birleştirebiliriz. Ardından form oluşturan React componenti, sonraki kullanıcı girişi üzerinde bu formda olanı da kontrol eder. Değeri React tarafından bu şekilde kontrol edilen bir giriş form elemanına kontrollü component denir.
 
-For example, if we want to make the previous example log the name when it is submitted, we can write the form as a controlled component:
+Örneğin, bir önceki örnekte, name değerinin yazılıp submit edildiğinde name i alert ile yazdırmak istiyorsak, formu kontrollü bir component olarak oluşturabiliriz:
```

Comment:
name derken addan bahsettiğin aşikar. Zaten yazının devamında da addan bahsediyorsun ("Örneğin, adın büyük harflerle yazılmasını isteseydik," kısmında).  

Bu nedenle:
name i alert ile yazdırmak istiyorsak
yerine 
adı ekranda mesaj olarak basmak istiyorsak
şeklinde çevirebilirsin.

Reply:
Bazı teknik terimlerin çevrilmesinden emin değilim. Alert fonksiyonu ve name özniteliği gibi. Sana hak vermekle birlikte burada bir ikilemde kaldım.

Reply:
Bence de her terim Türkçeye çevrilmemeli. Şahsen cümlenin yapısında çok fazla İngilizce kelime olunca içimden okurken Türkçe-Ingilizce kelime geçişlerinden dolayı bana garip gelmeye başlıyor.

İkilemde kalmamak için farklı kaynakları inceleyebilirsin. Mesela İspanyolca dokümantasyonda bu terimlerin nasıl ele alındığını inceleyebilirsin: https://es.reactjs.org/docs/forms.html
Ayrıca Slack kanalında bu durumu ele alabilirsin.